angular/core/components/poll/common/share/link_form/poll_common_share_link_form.haml

```diff
@@ -1,17 +1,10 @@
 %section
```

Comment:
I'd like to get to a point where we don't have a whole bunch of directives for different poll displays, and instead have a single bit of code which takes a poll search query somehow and displays the polls it finds

```
%poll_collection_card{model: "discussion", query: "closedPolls"}
%poll_collection_card{model: "group", query: "polls"}
```
etc.